curation/views.py

```diff
@@ -138,12 +141,62 @@ def _str_to_int(value):
         gene=gene,
     )
     if genomic_coords:
-        response['gene'] = genomic_coords.get("gene", None)
-        response['chr'] = genomic_coords.get("chr", None)
+        gene = genomic_coords.get('gene', None)
+        response['gene'] = gene
+        response['gene_id'] = normalize_gene(gene)['concept_id']
+        chr = genomic_coords.get("chr", None)
+        response['chr'] = chr
+        sequence_id = get_sequence_id(f'refseq:{chr}')
```

Comment:
This isn't necessary. Also, this would help account for cases where an Ensembl identifier was entered. 
```suggestion
        sequence_id = get_sequence_id({chr})
```